### DISCUSSION TITLE
Combating Spam?

### DISCUSSION THREAD

**Discussion by jordanlambrecht** [react-hook-form/react-hook-form#8588] [Q&A]
I use react-hook-form as a contact form on our website. We keep getting a ton of spam. Does anyone have experience with combating this? There's surprisingly little info out there on integrating RHF with reCaptcha, so it seems like that must not be the right route?

**Comment by bluebill1049** [comment]
Yea use reCapture should block most from the front end but you still may get spam from service layer as well. There are probably some good blog post on this topic.
